packages/esm-service-queues-app/src/patient-search/hooks/useScheduledVisits.ts

```diff
@@ -22,7 +22,7 @@ export function useScheduledVisits(patientUuid: string) {
     });
 
   const { data, error, isLoading } = useSWR<AppointmentsFetchResponse, Error>(
-    patientUuid ? appointmentsSearchUrl : null,
+    patientUuid ? [appointmentsSearchUrl, patientUuid] : null,
```

Comment:
Nice fix! I didn't know you can pass multiple keys into `useSWR`. 

https://swr.vercel.app/docs/arguments

A bit minor, but maybe refactor `fetcher` to take in `appointmentsSearchUrl` and `patientUuid` params to make it more clear?

Reply:
Yeah @chibongho , that makes it more clear, however `fetcher` is a variable in the same scope where `appointmentsSearchUrl` and `patientUuid` are declared, thus being directly used as 
```typescript
const fetcher = () =>
    openmrsFetch(appointmentsSearchUrl, {
      method: 'POST',
      signal: abortController.signal,
      headers: {
        'Content-Type': 'application/json',
      },
      body: {
        patientUuid: patientUuid,
        startDate: startDate,
      },
    });
```

Reply:
@usamaidrsk , a `fetcher` function is supposed to take the key as its argument, so you should write it as such. It's SWR's API so you need to follow their rules.